src/did/did.spec.ts

```diff
@@ -25,27 +25,37 @@ import { expect, use } from 'chai';
 import { Did, DidProofType } from './did';
 import { SignerIdentity } from '../contracts/signer-identity';
 import { TestUtils } from '../test/test-utils';
-import { accounts, useIdentity } from '../test/accounts';
+import {
+  accounts,
+  accountMap,
+  identities,
+  useIdentity,
+} from '../test/accounts';
 import {
   DigitalTwin,
   DigitalTwinOptions,
   Runtime,
 } from '../index';
 
+import { Vade } from '../../libs/vade';
+
 
 use(chaiAsPromised);
 
 (useIdentity ? describe : describe.skip)('DID Resolver', function test() {
-  this.timeout(600000);
+  this.timeout(60_000);
   let accounts0Identity: string;
   let accounts0Did: string;
   let runtimes: Runtime[];
+  let vade: Vade;
+  const vadeConfig = {}; // use default config but enable vade
+  const runtimeConfig = { useIdentity: true, vade: vadeConfig };
```

Comment:
You introduced some checks within tests (e.g.: `this.skip();` if vade is set), but currently the test in always runned with vade. Should we run it twice? Once with the old and one with the new logic?

Reply:
`this.skip` will be removed and `vade` will be enabled by default.